zhaquirks/tuya/ts004f.py

```diff
@@ -401,6 +401,86 @@ class TuyaSmartRemote004FSK(EnchantedDevice):
         (DOUBLE_PRESS, BUTTON_1): {ENDPOINT_ID: 1, COMMAND: DOUBLE_PRESS},
     }
 
+class TuyaSmartRemote004FSK_v2(EnchantedDevice):
```

Comment:
To make the _v2 class inherit from `TuyaSmartRemote004FSK`, change it to be like this:
```suggestion
class TuyaSmartRemote004FSK_v2(TuyaSmartRemote004FSK):
```